src/app/groups/page.tsx

```diff
@@ -1,5 +1,8 @@
 import { GroupsPage } from '@/components/page/GroupsPage';
+import { fetchGroupList } from '@/lib/api/data';
+import { IGroup } from '@/types/definition';
 
-export default function Groups() {
-  return <GroupsPage />;
+export default async function Groups() {
+  const groups: IGroup[] = await fetchGroupList();
```

Comment:
@kotaaaa @kanta1207 

I would like to discuss with you about the approach of fetching data from API.

I guess making the top-level component async may cause a delay in rendering time, which can be avoided. Instead, for the group collection page, I recommend making only the group card area async and using React suspend with a skeleton component. This will ensure that the entire page is displayed quickly by the browser, even while fetching data from the API. 

The same approach can be applied to the food collection page, thus it might be better to handle it in a separate ticket.

Reply:
I agree with @uskayyyyy 's opinion, 
- we should not use async function on the top-level component as much as possible.
- Use React suspend with a skeleton component

And @kanta1207  created this issue #157, we can handle there.

And I thought this modification can be done at next related PR.

AS IS
```
export default async function Groups() {
  return <GroupsPage />;
}
```

TO BE
```
export default function Groups() {
  return <GroupsPage />;
}
```